h2o-docs/src/product/data-science/gbm-faq/reproducibility.rst

```diff
@@ -223,3 +223,18 @@ Reproducibility
  8. Use the selected IP address to connect to H2O from R, Python, or Flow and re-train the model you are interested in reproducing. 
    
   **Note**: If you are using Flow, this means you must open Flow in a new browser with the IP and port selected.
+
+- **Why is the order of nodes (computations) so crucial for reproducibility?**
+ 
+  Digital computers are not able to work with real numbers precisely. The range of real numbers is infinite. However, computers can store only finite numbers with limited precision (usually in binary format). This causes several problems - some numbers are not precise in general and have to be rounded, and it also affects basic computation methods such as addition, multiplication, and division.
+ 
+  For example, number 0.1 is 0.1000000000000000055511151231257827021181583404541015625 in 64-bit floating point format. Also, adding 0.2 to 0.1 is not equal to 0.3 in this format, but 0.30000000000000004.
+
+  Performing floating-point arithmetic could cause a round-off (or rounding) error - a difference between the result produced by a given algorithm using exact arithmetic and the result generated by the same algorithm using finite-precision. (More information available here: https://en.wikipedia.org/wiki/Round-off_error.)
+
+  The error could be cumulated, and the laws of algebra do not necessarily hold. For example, algorithms based on unstable computations can generate various output for various inputdue to a cumulation of error, and associativity is not guaranteed in floating point.
```

Comment:
There is a typo here: should be `input due to` (missing space)

Reply:
Thank you @meganjkurka!